doc/man3/SSL_CTX_set_security_level.pod

```diff
@@ -106,6 +106,17 @@ versions below 1.2 are not permitted.
 Security level set to 256 bits of security. As a result RSA, DSA and DH keys
 shorter than 15360 bits and ECC keys shorter than 512 bits are prohibited.
 
+=item B<Exception ECC>
+
+256 bits are the minimum for ECC keys even at security level 2 and lower
+because, for example, P-224 is not enabled on default. Therefore, to use
```

Comment:
This is a little bit confusing I think. I'd say something like `By default no ECC groups smaller than 256 bits are enabled. Therefore, to use shorter keys than 256bits, the default groups list must be changed as well in addition to setting a security level allowing them.`

Reply:
Your suggestion is nice, I like it. I see the difference. To keep consistency, the same can be done for DH: `By default, no DH keys smaller than 1024 bits are allowed. Therefore, to use shorter keys, an application defined security callback is required.` However, I do not see what was confusing. To be honest, I do not like that whole current level explanation. It was very difficult for me to understand/extract the differences/changes of each level at first glance. In a project of mine, I had to explain it and [used a table](https://www.traud.de/voip/White_Paper.pdf) (page 4) instead.